pkg/rpcservice/logs_tail.go

```diff
@@ -0,0 +1,199 @@
+package rpcservice
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/stripe/stripe-cli/pkg/logtailing"
+	"github.com/stripe/stripe-cli/rpc"
+)
+
+// ITailer enables mocking a tailer object in tests
+type ITailer interface {
+	Run(context.Context) error
+}
+
+var createTailer = func(cfg *logtailing.Config) ITailer {
+	return logtailing.New(cfg)
+}
+
+// LogsTail returns a stream of API logs
+func (srv *RPCService) LogsTail(req *rpc.LogsTailRequest, stream rpc.StripeCLI_LogsTailServer) error {
```

Comment:
if `StripeCLI_LogsTailServer` an auto generated name? We don't usually use snake case in Go. `StripeCLILogsTailServer` would be better.

Reply:
I'm confused at the signature of this function - what's the entry point for clients?

In your tests in go, you do `client.LogsTail(ctx, &rpc.LogsTailRequest{})` and in javascript you pass `client.LogsTail(req)`. 

Seems like the same function is called differently, never with `stream`. Can you clarify?

Reply:
- Yes, `StripeCLI_LogsTailServer` is autogenerated so I can't change it.  The format is `<service name>_<method name>Server`.
- I can see the confusion. The signatures are different because they are not the same method. `RPCService#LogsTail` runs on the gRPC server and is completely transparent to clients. On the other hand, `StripeCLIClient#LogsTail` is the entry point for gRPC clients and is a different method entirely.